### PR TITLE
Adding Miraheze icon

### PR DESCRIPTION
**Pull request by pisceskaze**
miraheze

**Issue:** None

**Similarweb rank:**
https://www.similarweb.com/website/miraheze.org/ - `#9,669`

### Checklist
  - [X] I updated the JSON data in `_data/simple-icons.json`
  - [X] I optimized the icon with SVGO or SVGOMG
  - [X] The SVG `viewbox` is `0 0 24 24`

### Description
The logo was from a non-profit MediaWiki farm services provider, and no issue was created since there is no need to create one for this.

### CODE REVIEW

**Comment by pisceskaze** [comment]
oh my god, not again, the lint doesn't seems to know that `#fffc00` was a real color

**Comment by LitoMore** [comment]
@pisceskaze Use uppercase.

**Comment by pisceskaze** [comment]
Got it. It just weird that the hex color I chosen return a lint test failed.

**Comment by pisceskaze** [comment]
I finally know that I should write it UPPERCASE.

File: _data/simple-icons.json [hunk 1]
```diff
@@ -7643,6 +7643,11 @@
             "hex": "30B980",
             "source": "https://minutemailer.com"
         },
+        {
+            "title": "Miraheze",
+            "hex": "FFFC00",
+            "source": "https://commons.wikimedia.org/wiki/File:Miraheze-Logo.svg"
```
Comment (adamrusted) [inline _data/simple-icons.json:7649]:
```suggestion
            "source": "https://miraheze.org"
```

**Review by adamrusted — APPROVED** [review]
Icon is a perfect match, thank you @pisceskaze 🎉
I'll just make a quick update to the source, as an SVG icon is available on their homepage.

**Pull request merged**